youtube_dl/extractor/medialaan.py

```diff
@@ -193,7 +193,7 @@ def get(container, item):
             sso = get('vod', 'gigyaDatabase') or 'vtm-sso'
 
             data = self._download_json(
-                'http://vod.medialaan.io/api/1.0/item/%s/video' % vod_id,
+                'https://vod.medialaan.io/api/1.0/item/%s/video' % vod_id,
```

Comment:
I don't see any indication of this being accessed by browser over https. At the time of writing browser was accessing it over http. Provide example URL with such behavior.